<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>高校人员管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/img/image/icon.png</normaloff>:/img/image/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>671</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>901</width>
      <height>671</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>学生</string>
     </attribute>
     <widget class="QTableView" name="student_tableView">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>3</y>
        <width>621</width>
        <height>631</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="student_nameEdit">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>10</y>
        <width>221</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MiSans</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>31</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>60</y>
        <width>31</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>性别</string>
      </property>
     </widget>
     <widget class="QComboBox" name="student_genderComboBox">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>60</y>
        <width>221</width>
        <height>34</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>110</y>
        <width>31</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>出生</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="student_birthDateEdit">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>110</y>
        <width>221</width>
        <height>34</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="student_majorEdit">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>160</y>
        <width>221</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MiSans</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>160</y>
        <width>31</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>专业</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="student_classNameEdit">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>210</y>
        <width>221</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MiSans</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>210</y>
        <width>31</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>班级</string>
      </property>
     </widget>
     <widget class="QPushButton" name="student_addBtn">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>260</y>
        <width>261</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>添加学生</string>
      </property>
     </widget>
     <widget class="QPushButton" name="student_saveBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>260</y>
        <width>125</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="student_returnBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>765</x>
        <y>260</y>
        <width>125</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>返回添加</string>
      </property>
     </widget>
     <zorder>student_addBtn</zorder>
     <zorder>student_tableView</zorder>
     <zorder>student_nameEdit</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>student_genderComboBox</zorder>
     <zorder>label_3</zorder>
     <zorder>student_birthDateEdit</zorder>
     <zorder>student_majorEdit</zorder>
     <zorder>label_4</zorder>
     <zorder>student_classNameEdit</zorder>
     <zorder>label_5</zorder>
     <zorder>student_saveBtn</zorder>
     <zorder>student_returnBtn</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>教师</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>查询</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionHome"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menuSetting"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionHome">
   <property name="text">
    <string>导入学生</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>student_nameEdit</tabstop>
  <tabstop>student_genderComboBox</tabstop>
  <tabstop>student_birthDateEdit</tabstop>
  <tabstop>student_majorEdit</tabstop>
  <tabstop>student_classNameEdit</tabstop>
  <tabstop>student_addBtn</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>student_tableView</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
